proto/tendermint/abci/types.proto

```diff
@@ -319,6 +333,14 @@ message ResponseProcessProposal {
   tendermint.types.ConsensusParams consensus_param_updates = 5;
 }
 
+message ResponseExtendVote {
+  bytes vote_extension = 1;
+}
+
+message ResponseVerifyVoteExtension {
+  bool accept = 1;
```

Comment:
We might want to consider making this an enumeration.

```proto
enum VerifyStatus {
    UNKNOWN = 0;
    ACCEPT  = 1;
    REJECT  = 2;
}
```

or words to that effect. This forces the field to be set explicitly, so we can tell whether the application actually rejected the extension or just didn't care. Maybe this doesn't matter, but I'm thinking about network upgrades where the app goes from a version that doesn't use extensions to one that does—and how the upgrade occurs.

Reply:
I see your point. I think I agree with this, however, if we go down this path, we'd need to be coherent and change other Accept/Reject booleans in the spec (e.g., `ResponseProcessProposal`).
Let's leave it as it is in this PR. I just created a separate issue in ABCI++ board to change this consistently in all parts of ABCI++

Reply:
Yes, I agree that if we do decide we want this, we should be systematic about it. Definitely not a blocker for this PR.